observability/logging/logging-6.0/log6x-loki.adoc

```diff
@@ -0,0 +1,46 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-loki"]
+= Storing logs with LokiStack
+include::_attributes/common-attributes.adoc[]
+:context: logging-6x
+
+toc::[]
+
+You can configure a `LokiStack` CR to store application, audit, and infrstructure-related logs. 
+
+== Installation
+Partial includes of modules/logging-loki-cli-install.adoc & logging-loki-gui-install.adoc once https://github.com/openshift/openshift-docs/pull/77407/ is merged.
```

Comment:
@libander, do we want to include these installation modules or just reference them.
Maybe, we can use a prerequites section for the assembly to mention:
You have installed the Loki Operator by using the CLI or web console. For more details, see cross-references.

Reply:
xrefs work. :)